fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -304,12 +310,13 @@ class GlobalTagThrottlerImpl {
 	// Returns the desired cost for a storage server, based on its current
 	// cost and throttling ratio
 	Optional<double> getLimitingCost(UID storageServerId) const {
-		auto const throttlingRatio = tryGet(throttlingRatios, storageServerId);
+		auto const ssInfo = tryGet(ssInfos, storageServerId);
+		auto const throttlingRatio = ssInfo.present() ? ssInfo.get().throttlingRatio : Optional<double>{};
```

Comment:
[Optional] Might be useful to specify the type here (instead of auto) as the type will be informative (to future readers who might not realize it's `Optional`, for example) without being too verbose.